addons/html_builder/static/src/snippets/snippet_viewer.js

```diff
@@ -105,7 +105,10 @@ export class SnippetViewer extends Component {
             const strMatches = (str) =>
                 str.toLowerCase().includes(this.props.state.search.toLowerCase());
```

Comment:
(str) =>
               str ?  str.toLowerCase().includes(this.props.state.search.toLowerCase()) : false ;